bindings/python/libmata.pxd

```diff
@@ -240,11 +240,11 @@ cdef extern from "mata/nfa.hh" namespace "Mata::Nfa":
         void unify_initial()
         void unify_final()
         COrdVector[Symbol] get_used_symbols()
-        void add_trans(CTrans) except +
-        void add_trans(State, Symbol, State) except +
-        void remove_trans(CTrans) except +
-        void remove_trans(State, Symbol, State) except +
-        bool has_trans(State, Symbol, State)
+        void add(CTrans) except +
+        void add(State, Symbol, State) except +
+        void remove(CTrans) except +
+        void remove(State, Symbol, State) except +
+        bool contains(State, Symbol, State)
```

Comment:
The Python binding has to be modified. This will not work as there would be multiple `add` operations, for example (one to add new state, other to add new transition etc.)

I would suggest reverting all changes to Python binding and @tfiedor or I will have to modify the Python binding manually.

Reply:
Okay, I reverted all changes in Python binding. Please do it manually.